native_client/definitions.mk

```diff
@@ -0,0 +1,27 @@
+TARGET ?= host
+TFDIR  ?= ../../tensorflow
+CXX    ?= c++
+PREFIX ?= /usr/local
+
+ifeq ($(TARGET),host)
+TOOLCHAIN :=
+CFLAGS    := `pkg-config --cflags sox`
+LDFLAGS   := `pkg-config --libs sox`
+endif
+
+ifeq ($(TARGET),rpi3)
+TOOLCHAIN   ?= ${TFDIR}/bazel-$(shell basename "${TFDIR}")/external/GccArmRpi/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/arm-linux-gnueabihf-
+RASPBIAN    ?= ../multistrap-raspbian-jessie
+CFLAGS      := -isystem $(RASPBIAN)/usr/include -isystem $(RASPBIAN)/usr/include/arm-linux-gnueabihf
+LDFLAGS     := -Wl,-rpath-link,$(RASPBIAN)/lib/arm-linux-gnueabihf -Wl,-rpath-link,$(RASPBIAN)/usr/lib/arm-linux-gnueabihf/ $(RASPBIAN)/usr/lib/arm-linux-gnueabihf/libsox.so
```

Comment:
I had to switch from `-lsox` there to `libsox.so` because the linker was hitting some weird bug